prediction_market_agent/agents/safe_guard_agent/safe_utils.py

```
@@ -0,0 +1,89 @@
+from eth_account.messages import defunct_hash_message
+from prediction_market_agent_tooling.config import APIKeys, RPCConfig
+from prediction_market_agent_tooling.gtypes import HexBytes
+from prediction_market_agent_tooling.loggers import logger
+from safe_eth.eth import EthereumNetwork
+from safe_eth.safe import Safe
+from safe_eth.safe.api.transaction_service_api import TransactionServiceApi
+from safe_eth.safe.api.transaction_service_api.transaction_service_api import (
+    EthereumNetwork,
```

Ruff (0.8.2)

Redefinition of unused `EthereumNetwork` from line 5

(F811)